definitions/features/tar.rb

```diff
@@ -0,0 +1,86 @@
+class Features::Tar < ForemanMaintain::Feature
+  metadata do
+    label :tar
+    preparation_steps { Procedures::Packages::Install.new(:packages => %w[tar]) }
+  end
+
+  # Run system tar
+  # @param [Hash] options tar configuration
+  # @option options [String] :archive archive name
+  # @option options [String] :command ('create') tar operation command
+  # @option options [Array] :exclude dirs or files to exclude
+  # @option options [String] :listed_incremental .snar file to do incremental backup
+  # @option options [String] :transform sed expression to transform the filenames
+  # @option options [String] :volume_size size of tar volume (will try to split the archive when set)
+  # @option options [String] :directory change to directory DIR
+  # @option options [Boolean] :multi_volume create/list/extract multi-volume archive
+  # @option options [Boolean] :overwrite overwrite existing files when extracting
+  # @option options [Boolean] :gzip filter the archive through gzip
+  # @option options [String] :files (*) files to operate on
+  def run(options = {})
+    volume_size = options.fetch(:volume_size, nil)
+    validate_volume_size(volume_size) unless volume_size.nil?
+
+    tar_command = ['tar']
+    tar_command << '--selinux'
+    tar_command << '--overwrite' if options[:overwrite]
+    tar_command << '--gzip' if options[:gzip]
+    tar_command << "--#{options.fetch(:command, 'create')}"
+    tar_command << "--file=#{options.fetch(:archive)}"
+
+    if volume_size
+      split_tar_script = default_split_tar_script
+      tar_command << "--tape-length=#{volume_size}"
+      tar_command << "--new-volume-script=#{split_tar_script}"
```

Comment:
I'll need this without --tape-length (see [here](https://git.io/vxxh7)), can you make --volume-script a separate option? I can do it in my PR, but will make things easier if you change here.